frontend/src/app/modules/boards/board/inline-add/board-inline-add-autocompleter.component.ts

```diff
@@ -117,12 +157,13 @@ export class BoardInlineAddAutocompleterComponent implements AfterViewInit {
   }
 
   public addWorkPackageToQuery(workPackage?:WorkPackageResource) {
+    
     if (workPackage) {
       this.schemaCacheService
         .ensureLoaded(workPackage)
         .then(() => {
           this.onReferenced.emit(workPackage);
-          this.ngSelectComponent.close();
+          this.ngSelectComponent.ngSelectInstance.close();
```

Comment:
It is not the default behavior of the ng-select to close itself when an option is selected?